openedx/adg/lms/applications/tests/conftest.py

```diff
@@ -94,9 +105,58 @@ def expired_course_overview(current_time):
     )
 
 
-@pytest.fixture()
-def test_user():
+@pytest.fixture(name='test_user')
+def testing_user():
     """
     Fixture which returns a user
     """
-    return UserFactory()
+    return UserFactory(username=USERNAME, password=PASSWORD)
+
+
+@pytest.mark.django_db
+@pytest.fixture(name='user')
+def user_with_application_hub_and_extended_profile_fixture(test_user):
```

Comment:
If we are naming the above user `test_user`, for consistency, this should also be called `test_user_with_application_hub_and_extended_profile`. Names of fixtures should be consistent.

How would a reader differentiate between `test_user` and `user` once they are added as arguments in a test function? Name should be more descriptive.

Reply:
It is being used at a lot of different places as `user`. That is why I didn't change it's fixture name

Reply:
I'm removing `test_user` fixture altogether, and using `user` fixture everywhere.